docs/react/reference/useMutation.md

```diff
@@ -43,7 +43,8 @@ mutate(variables, {
 **Options**
 
 - `mutationFn: (variables: TVariables) => Promise<TData>`
-  - **Required**
+  - Optional
+  - Defaults to the global mutation configuration `mutationFn`, which is `undefined` by default.
```

Comment:
Hmm, we will die at runtime if the default of `undefined` is used. To align with the useQuery docs, I would say:

```suggestion
  - **Required, but only if no default mutation function has been defined**
```

Reply:
Awesome suggestion, nice one 👍